app/components/PolicyContainer.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+import logo from '../assets/full_logo.png';
+
+type ParsedLine = {
+  type:
+    | 'paragraph'
+    | 'heading1'
+    | 'heading2'
+    | 'heading3'
+    | 'listItem'
+    | 'break';
+  content: string;
+};
+
+type PolicyContainerProps = {
+  content: string;
+};
+
+export function PolicyContainer({ content }: PolicyContainerProps) {
+  const [parsedContent, setParsedContent] = useState<ParsedLine[]>([]);
+
+  // Simple markdown parser without external libraries
+  useEffect(() => {
```

Comment:
why not useMemo? if I am not missing something there are no side effects here. we are only computing `parsedContent` from the `content` and we want to memoize the computation not to do it on every render